workflows/sonataflow-callbackstatetimeouts-persistence/02-configmap_callbackstatetimeouts-props.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+data:
+  application.properties: |
+    kogito.service.url=http://callbackstatetimeouts
```

Comment:
Alternatively, we could use the svc discovery? `${kubernetes:services.v1/callbackstatetimeouts}`.

Reply:
This is also fine since it uses the internal DNS mounted in the pod. Just an observation if we need to test the discovery later.

Reply:
I've spent some time trying here but it looks like the service discovery is not working.
I have opened this https://issues.redhat.com/browse/KOGITO-9806  and assigned to you for a second look, in case something is missing on my test, but I think that it's bug. Take a second look please, then we see who can resolve it.

Reply:
@ricardozanini :point_up:

Reply:
added